### ISSUE TITLE
Train using my own image dataset.

### ISSUE THREAD

**Issue by ofsazib** [georgesung/traffic_sign_classification_german#2]
I want to train a new model with my own image set.Then how can I create train.p, test.p file?

**Comment by georgesung** [comment]
Basically, for all your images in your dataset, resize them to 32x32 RGB and make it a numpy array of shape [32, 32, 3]. Then put your images into a dictionary like such: {'features': [image0, image1, ...], 'labels': [integer_label0, integer_label1, ...]}. Do this for both your training data and test set.

**Comment by ofsazib** [comment]
@georgesung  Would have any paper for this project?

**Comment by georgesung** [comment]
No paper in particular, since this is a pretty generic image classification task, like classifying ImageNet or CIFAR-10 for example. If you look online for tutorials on how to classify CIFAR-10, MNIST, or ImageNet data, the process should be similar.
